admin/multiadd.php

```diff
@@ -17,55 +17,76 @@
-				<br/>
-				Each entry will automatically have the month incremented.
-				</p>
-				<form id="input_select" method="post" action="multiaddprocess.php">
-					<label for="series_name">Series</label>
-				  <?php echo $dropdown; ?>
-				  <br/>
-				  <label for="first_issue">First Issue</label>
-				  <input name="first_issue" type="text" maxlength="3" />
-				  <br/>
-					<label for="last_issue">Last Issue</label>
-					<input name="last_issue" type="text" maxlength="3" />
-					<br/>
-					<label for="original_purchase">Purchased When Released:</label>
-            		<input name="original_purchase" value="1" type="radio" />Yes
-            		<input name="original_purchase" value="0" type="radio" />No
-            		<br/>
-            		<label for="release_date">First Comic's Published Date</label>
-				  	<input name="release_date" type="text" maxlength="10" placeholder="YYYY-MM-DD"/>
-				  	<br/>
-					<input type="submit" name="submit" value="Submit" />
+				<?php
+					if ($login->isUserLoggedIn () == false) {
+						include(__ROOT__."/views/not_logged_in.php");
+						die ();
+					}
+				?>
+				<h2>Add a range of issues</h2>
+				<p>If the series had a regular monthly release, you may enter the published date of the first issue.</p>
+				<p>Each entry will automatically have the month incremented.</p>
+				<form id="input_select" method="post" action="multiaddprocess.php" class="container">
+					<div class="row">
+						<div class="col-xs-12 col-md-6">
+							<div class="form-group">
+								<label for="series_name">Series</label>
+						  	<?php echo $dropdown; ?>
+							</div>
+						</div>
+						<div class="col-xs-12 col-md-6">
+							<div class="form-inline">
+								<div class="form-group">
+							  	<label for="first_issue">First Issue</label>
+							  	<input name="first_issue" type="text" class="form-control" maxlength="3" size="3" />
+							  </div>
+								<div class="form-group">
+									<label for="last_issue">Last Issue</label>
+									<input name="last_issue" type="text" class="form-control" maxlength="3" size="3" />
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="row">
+						<div class="col-md-12 col-xs-12">
+							<div class="form-inline">
+								<div class="form-group form-radio">
+									<label for="original_purchase">Purchased When Released</label>
+									<fieldset>
+										<input name="original_purchase" id="original-yes" value="1" type="radio" /> <label for="original-yes">Yes</label>
+										<input name="original_purchase" id="original-no" value="0" type="radio" /> <label for="original-no">No</label>
+			          	</fieldset>
+			          </div>
+			          <div class="form-group">
+			            <label for="release_date">First Comic's Published Date</label>
+							  	<input name="release_date" type="date" class="form-control" maxlength="10" placeholder="YYYY-MM-DD"/>
```

Comment:
Input type="date" is awesome! In most modern browsers it will auto add a little dropdown calendar, and it will save the data by default in the standard YYYY-MM-DD format without having to make the user write out the European standard date format.